go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/hashicorp/go-version v1.6.0
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	github.com/kubernetes-csi/csi-proxy/client v1.0.1
-	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.1.0
+	github.com/kubernetes-csi/external-snapshotter/client/v6 v6.2.0
```

Comment:
we need to use v6.1.0 since WCP CRDs will be deployed with v6.1.0
@deepakkinni please correct if not so.

Reply:
Yes, 6.1.0

Reply:
Updated code to use v6.1.0.